core/src/main/scala/com/databricks/labs/remorph/intermediate/expressions.scala

```diff
@@ -193,6 +211,14 @@ trait StarOrAlias
 
 case class Alias(expr: Expression, name: Id) extends Unary(expr) with StarOrAlias {
   override def dataType: DataType = expr.dataType
+
+  override def makeCopy(newArgs: Array[AnyRef]): Expression = {
```

Comment:
`TreeNode.makeCopy` docs say:
```
Must be overridden by child classes that have
   * constructor arguments that are not present in the productIterator.

```
I don't know exactly why only 1 arg of the `Alias` ctor is seen by the `productIterator` (maybe because it's not a `val` ?), but it's what happens.

This is tested via existing test cases in `SnowflakeToDatabricksTranspilerTest`, which crash without this bug fix.